typedspark/_utils/create_dataset.py

```diff
@@ -53,6 +55,36 @@ class Person(Schema):
     Any columns in the schema that are not present in the data will be
     initialized with ``None`` values.
```

Comment:
Maybe good to add to the docstring example usages? Especially now that it takes two forms of input (dict & list)?